src/core/Slider/Slider.tsx

```diff
@@ -0,0 +1,638 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { useMergedRefs } from '../utils/hooks/useMergedRefs';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/slider.css';
+import { Tooltip } from '../Tooltip';
+import { Placement } from 'tippy.js';
+import { StylingProps } from '../utils/props';
+
+/** Specifies domain of allowable values. */
+export interface ValueDomain {
```

Comment:
We use `type` instead of `interface`. This is defined in our STYLEGUIDE too.

Reply:
removed